src/Relation/Nullary/Decidable/Core.agda

```diff
@@ -69,9 +68,8 @@ module _ {A : Set a} where
 
 -- Given an irrelevant proof of a decidable type, a proof can
 -- be recomputed and subsequently used in relevant contexts.
-recompute : Dec A → .A → A
-recompute (yes a) _ = a
-recompute (no ¬a) a = ⊥-elim (¬a a)
+recompute : Dec A → Recomputable A
+recompute = Reflects.recompute ∘ proof
```

Comment:
Should this be renamed to `dec⇒recomputable` on the model of `Relation.Unary.Consequences`?

Reply:
Depends whether we see `Dec` as a predicate or a datatype
e.g. we use the name `toVec` rather than `list⇒vec`.

Reply:
Nice analysis! (as with our other discussion on #2059 regarding `toSum`/`toDec` etc.)

I guess this means that, here at least, I have an inflection towards 'property' over 'datatype' (hence also my raising of #2091 )... but whether that's an 'innate' choice, or conditioned by/contingent on all of this happening under the aegis of `Nullary`, I'm not sure.

Cf. `Maybe` as a datatype constructor; `WeaklyDecidable` as its `Nullary` property incarnation...

Reply:
Sticking with `recompute` (for the time being)...